mne/minimum_norm/tests/test_inverse.py

```diff
@@ -1097,6 +1099,41 @@ def test_apply_mne_inverse_epochs():
             inverse_operator, 1.)
 
 
+@pytest.mark.slowtest
+@testing.requires_testing_data
+@pytest.mark.parametrize('return_generator', (True, False))
+def test_apply_inverse_tfr(return_generator):
+    """Test applying an inverse to time-frequency data."""
+    rng = np.random.default_rng(11)
+    n_epochs = 4
+    info = read_info(fname_raw)
+    inverse_operator = read_inverse_operator(fname_full)
+    freqs = np.arange(8, 10)
+    times = np.linspace(0, 1, int(round(info['sfreq'])))
+    data = rng.random((n_epochs, len(info.ch_names), freqs.size, times.size))
+    data = data + 1j * data  # make complex to simulate amplitude + phase
+    epochs_tfr = EpochsTFR(info, data, times=times, freqs=freqs)
+    epochs_tfr.apply_baseline((0, 0.5))
+    pick_ori = 'vector'
+
+    # test epochs
+    stcs = apply_inverse_tfr_epochs(
+        epochs_tfr, inverse_operator, lambda2, "dSPM", pick_ori=pick_ori,
+        return_generator=return_generator)
```

Comment:
can you check that if you average stcs in vector mode you get the same output as if you would apply_inverse on the average of EpochsTFR.data?

Reply:
Sure, you have to average over the epochs and the frequency dimension since we did away with the `apply_inverse_tfr` to `AverageTFR` objects though but if you do that it works.